backend/src/routes/report.js

```diff
@@ -0,0 +1,21 @@
+const controller = require("../controllers/report");
+const validators = require("../validators/report");
+const what = "report";
+
+const routes = [
+  {
+    method: "GET",
+    url: `/${what}/projects/:projectId`,
```

Comment:
I'm wondering how this route is going to work with other reports specific to this project?  From this route i would conclude there is only one report per project.